### ISSUE TITLE
Allow trailing inline comments on shellcheck disable directives

### ISSUE THREAD

**Issue by stdedos** [koalaman/shellcheck#1391]
#### For bugs
- Rule Id (if any, e.g. SC1000): X
- My shellcheck version (`shellcheck --version` or "online"): online
- [x] I read the issue's wiki page, e.g. https://github.com/koalaman/shellcheck/wiki/SC2086
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash

#shellcheck disable=SC2015 # Intentional
:
```

#### Here's what shellcheck currently says:

```sh
Line 3:
#shellcheck disable=SC2015 # Intentional
^-- SC1073: Couldn't parse this shellcheck directive. Fix to allow more checks.
                                      ^-- SC1072: Expected '=' after directive key. Fix any mentioned problems and try again.

```

#### Here's what I wanted or expected to see:

Nothing. I'd like that I can leave a comment why I'm disabling the check (intentional, non-worthy workaround, "can't find a better way" etc)

**Comment by matthewpersico** [comment]
What version of shellcheck are you using? I do this all the time with version 0.5.0, no issues. Try removing the space between # and your first word.

**Comment by stdedos** [comment]
> * My shellcheck version (shellcheck --version or "online"): online

I _personally_ use the Ubuntu Xenial one (lacking a better option without the compiling-from-sources complexity), but I have had that replicated in online.

Now it doesn't replicate in online, but maybe I have a duplicate issue which was fixed?
Or it was fixed without an issue number / with a duplicate issue? I don't know

**Comment by koalaman** [comment]
This should have been fixed in May 2017 in 5099ebf9b and not touched since. I'm also unable to reproduce it now.

I don't know if there's an encoding issue or similar that could have caused it, but if you're able to reproduce it again, please reopen. This should most definitely a useful feature that should be allowed

**Comment by stdedos** [comment]
One instance I did see it in the xenial package as SC1072; however online did not show it.

**Comment by koalaman** [comment]
This would indeed be better written as `# shellcheck disable=SC2015 # Intentional` rather than having shellcheck silently accept random words in the directive.
